graylog2-web-interface/src/logic/alerts/EventDefinitionPriorityEnum.ts

```diff
@@ -16,12 +16,13 @@
  */
 const EventDefinitionPriorityEnum = {
   LOW: 1,
-  NORMAL: 2,
+  MEDIUM: 2,
   HIGH: 3,
```

Comment:
Is there any need to add
```suggestion
  HIGH: 3,
  CRITICAL: 4,
```
here?

Reply:
Nice catch. Just added in latest commit.